src/src/main_gps-telemetry.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * @file main_telemetry.cpp
+ * @author Cassandra "ZZ Cat" Robinson (nicad.heli.flier@gmail.com)
+ * @brief This file demonstrates how to use CRSF for Arduino to send GPS telemetry to an ExpressLRS RC receiver.
+ * @version 0.4.0
+ * @date 2023-07-25
+ *
+ * @copyright Copyright (c) 2023, Cassandra "ZZ Cat" Robinson. All rights reserved.
+ *
+ * @section License GNU General Public License v3.0
+ * This main file is a part of the CRSF for Arduino library.
+ * CRSF for Arduino is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * CRSF for Arduino is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CRSF for Arduino.  If not, see <https://www.gnu.org/licenses/>.
+ * 
+ */
+
+#if defined(ARDUINO) && defined(PLATFORMIO)
+#include "Arduino.h"
+
+#include "CRSFforArduino.h"
+
+/* Configuration Options. */
+#define VIEW_RC_CHANNELS         0 // Set VIEW_RC_CHANNELS to 1 to view the RC channel data in the serial monitor.
+#define GENERATE_RANDOM_GPS_DATA 0 // Set GENERATE_RANDOM_GPS_DATA to 1 to generate random GPS telemetry data.
+
+uint32_t timeNow = 0;
+
+/* Initialise the GPS telemetry data with default values. */
+float latitude = -41.18219482686493F; // Latitude is in decimal degrees.
+float longitude = 174.9497131419602F; // Longitude is in decimal degrees.
+float altitude = 100.0F;              // Altitude is in centimetres.
+float speed = 500.0F;                 // Speed is in cm/s
+float groundCourse = 275.8F;          // Ground Course is in degrees.
+uint8_t satellites = 4;
+
+CRSFforArduino crsf = CRSFforArduino(&Serial1);
+
+void setup()
+{
+#if VIEW_RC_CHANNELS > 0
+    Serial.begin(115200);
+    while (!Serial)
+    {
+        ;
+    }
+
+    Serial.println("GPS Telemetry Example");
+#endif
+
+    /* Initialise CRSF for Arduino */
+    crsf.begin();
```

Comment:
Might be an idea that I update this to include error handling like what I did over in `main_rc.cpp`...

```c++
// Initialize the CRSFforArduino library.
if (!crsf.begin())
{
    Serial.println("CRSF for Arduino initialization failed!");
    while (1)
    {
        ;
    }
}
```

That way, if any errors crop up, I will know what's going on.